packages/terra-badge/src/_mixins.scss

```diff
@@ -2,3 +2,34 @@
   background-color: map-get($color-map, background-color);
   color: map-get($color-map, color);
 }
+
+$default-theme: 'consumer' !default;
+$bundled-themes: 'consumer' 'ocs' !default;
+
+@mixin terra-consumer-theme {
+  @if index($bundled-themes, 'consumer') {
```

Comment:
should we wrap this in `:global` in the event someone accidentally calls this within `:local`